test/commands.test.js

```diff
@@ -1834,17 +1832,17 @@ test('[commands.operations.confirmChangeset] acccepted', function(assert) {
     },
     next: function() {
       assert.pass('success');
-      prompt.confirm.restore();
+      prompt.YesOrNo.restore();
+      assert.end();
```

Comment:
Without the assert.end, we would get indeterministic tape failures as it does not do well with asynchronous code.